pelican/plugins/show_source/show_source.py

```diff
@@ -1,59 +1,78 @@
 import logging
 import os
-from urllib.parse import urljoin
+import six
```

Comment:
We don't support Python 2.7 any longer, so I don't think we need the `six` library or the conditional import that follows below.